src/lib/Icons.ts

```diff
@@ -241,6 +243,7 @@ type IconName = Extract<
   | 'images'
   | 'info-circle'
   | 'language'
+  | 'level-down'
```

Comment:
De naam in Font Awesome is `arrow-turn-down`. Waarom gebruik je die naam niet?

Ik zie dat Karel in het design `arrow-turn-down` 90 graden naar links draait, maar je kunt dan toch net zo goed `arrow-turn-right` gebruiken? Dan hebben we (nog) geen support voor rotatie nodig.

Reply:
Idd. Ik wist niet dat die ook bestond. Zal dat doen.